src/molecules/Toggle/index.jsx

```diff
@@ -0,0 +1,45 @@
+export class Toggle extends React.Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+      isActive: false
+    };
+    this.toggleSwitch = this.toggleSwitch.bind(this);
+  };
+
+  componentDidMount(){
```

Comment:
We should move your props declaration inside of the `render()` function.

I have an example of this for reference at `src/organisms/GlobalHeader/index.jsx`